examples/cpp/continuousquery/Order.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Order.hpp"
+
+#include <geode/PdxReader.hpp>
+#include <geode/PdxWriter.hpp>
+
+namespace continuousquery {
+
+void Order::fromData(PdxReader& pdxReader) {
+  order_id_ = static_cast<uint32_t>(pdxReader.readLong(ORDER_ID_KEY_));
```

Comment:
Why muddy the example with types we have to cast?

Reply:
Yeah, kinda silly.  Fixed types, and fixed in other copies of Order.cpp/Order.hpp in examples.